.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for contributing a pull request! Please try to provide as much detail as possible to help the reviewer understand your work.
+You can also add the appropriate labels to describe the topic of the pull request and the type of changes you're making.
+-->
+
+### Motivation or Problem
+A clear and concise description of what what you're trying to fix or improve. Please reference any issues that this addresses.
+
+### Description of Changes
+A clear and concise description of what what you've changed or added.
+
+### Testing
+A clear and concise description of testing that you've done or plan to do.
+
+### Reviewer Tips
+Any suggestions or notes for the reviewer.
```

Comment:
the reviewer tips seems a little vague. 'Suggestions for how the reviewer can verify this PR is good'